pages/pil/application/views/exemption-data.jsx

```diff
@@ -14,18 +14,10 @@ class ExemptionData extends Component {
               <div className="govuk-grid-column-two-thirds">
                 <dl className="exemption">
                   <dt><Snippet>certificate.exemption</Snippet>:</dt>
-                  <dd>
-                    <ul className="modules">
-                      {
-                        exemption.modules.map((module, key) => (
-                          <li key={key}>
-                            {module.module}
-                            {module.exemptionDescription}
-                          </li>
-                        ))
-                      }
-                    </ul>
-                  </dd>
+                  <dd>{ exemption.modules[0].module }</dd>
+
+                  <dt><Snippet>certificate.reason</Snippet></dt>
```

Comment:
This content key probably wants renaming when we do the refactor...

Reply:
yeah